packages/@aws-cdk/integ-runner/lib/cli.ts

```diff
@@ -32,6 +32,7 @@ export async function main(args: string[]) {
     .option('disable-update-workflow', { type: 'boolean', default: false, desc: 'If this is "true" then the stack update workflow will be disabled' })
     .option('app', { type: 'string', default: undefined, desc: 'The custom CLI command that will be used to run the test files. You can include {filePath} to specify where in the command the test file path should be inserted. Example: --app="python3.8 {filePath}".' })
     .option('test-regex', { type: 'array', desc: 'Detect integration test files matching this JavaScript regex pattern. If used multiple times, all files matching any one of the patterns are detected.', default: [] })
+    .option('language', { type: 'array', default: ['javascript', 'typescript'], desc: 'The language presets to run tests for. Only javascript and typescript are supported for now.' })
```

Comment:
```suggestion
    .option('language', { type: 'array', choices: ['javascript', 'typescript'], default: ['javascript', 'typescript'], desc: 'The language presets to run tests for.' })
```

Reply:
We might want to check if  adding `defaultDescription: 'all supported languages'` provides a better ux